src/classes/PS_IntegrationServices.cls

```diff
@@ -0,0 +1,48 @@
+/*
+    Copyright (c) 2020, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group HGE
+* @group-content ../../ApexDocContent/GiftEntry.htm
+* @description Provides a public interface to the Payment Services Integration Package
+*/
+global class PS_IntegrationServices {
+    
+    /*******************************************************************************************************
+    * @description Provides an endpoint for the Payment Services Integrations package
+    * to call when a Customer registers, and pass parameters containing the secrets
+    * needed for calls to Payment Services endpoints(Tokenize, Purchase) and creating JWTs
+    * @param secrets contains secrets, such as the api key for creating a JWT
+    */
+    global static void setSecrets(Map<String, List<Map<String, String>>> secrets) {
+
```

Comment:
Maybe add in a TO-DO here to make it clear that this will be implemented in the future? Especially if we are going to merge it into master, let's be clear about our intent there

Reply:
Also - maybe want to re-think the name of this. Not everything is a secret, and it's more like "setOnboardingData" or "setElevateConfig" or something like that